tutorial/westeros/westeros_baseline.ipynb

```diff
@@ -1322,7 +1363,7 @@
    "source": [
     "# Only show a subset of technologies in the following plots;\n",
     "# e.g. exclude \"bulb\" and \"grid\"\n",
-    "rep.set_filters(t=[\"coal_ppl\", \"wind_ppl\"])\n",
+    "rep.set_filters(t=[\"coal_ppl\", \"wind_ppl\"], ya=[700, 710, 720])\n",
```

Comment:
@glatterf42 and @khaeru, I needed to filter the years explicitly here, because the plotting module does not show the values in historical years, which in this case is 690. Ideally, the plot should join `historical_activity` (for year 690) to `ACT` (for years 700, 710, and 720). If not possible, the plotting should not print historical year (690) without showing any value for that, which is the default behavior now.

Reply:
@glatterf42 , if we don't want to address this issue through this PR, I would say this is ready to be merged.

Reply:
Thanks @behnam-zakeri. I don't think we should use this PR for that issue. I'll create a new issue based on your comment, please update that to accurately describe the behaviour you would like to see :)